OpenShift/helm/nexus-repository-manager/templates/deployment-statefulset.yaml

```diff
@@ -59,9 +59,9 @@ spec:
         - name: nexus
           image: {{ .Values.nexus.imageName }}:{{ .Values.nexus.imageTag }}
           imagePullPolicy: {{ .Values.nexus.imagePullPolicy }}
-          {{- if .Values.deployment.terminationGracePeriodSeconds }}
-          terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
-          {{- end }}
+          # {{- if .Values.deployment.terminationGracePeriodSeconds }}
+          # terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
+          # {{- end }}
```

Comment:
please delete rather than comment out the code

Reply:
to @adrianpowell 's point, maybe this code can be left, and the entry in values.yaml can remain commented away?  I think that's enough for my purposes as well.